### ISSUE TITLE
Angular 4.1.0 + TyepScript : Can't bind to 'infiniteScrollDistance' since it isn't a known property of 'div' 

### ISSUE THREAD

**Issue by Jvick13** [orizens/ngx-infinite-scroll#222]
When I tried to compile using AOT I got these error. 

I put the import of InfiniteScrollModule in the app.module.

**Comment by orizens** [comment]
this error is not cause because of AOT. 
try following the demo code.

**Comment by Jvick13** [comment]
Sorry, i did't explain with calrity what happen.
Is the first time that I tried to use ngx-infinite-scroll. I follow the steps,  installing and import the InfiniteScrollModule to the main module, I tried to use this code in another module of the poject but when I used  **ng build --prod --aot** this error appear:

`Can't bind to 'infiniteScrollDistance' since it isn't a known property of 'div'. ("<div
		    infiniteScroll
		    [ERROR ->][infiniteScrollDistance]="2"
		    [infiniteScrollThrottle]="50"
		    (scrolled)="onScroll()">
")` 

This is the code that I use
`<div class="search-results"
	infiniteScroll
	[infiniteScrollDistance]="2"
	[infiniteScrollThrottle]="50"
	(scrolled)="onScroll()">
</div>`

**Comment by gregduffield** [comment]
Had the same issue, make sure that you are also exporting the Module as well as Importing in your module.ts

**Comment by Jvick13** [comment]
@gregduffield  These is the app.module.ts (I omit the other modules)

`import { NgModule } from '@angular/core';
import { BrowserModule } from '@angular/platform-browser';
import { InfiniteScrollModule } from 'ngx-infinite-scroll';
import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
import { AppComponent } from './app';
import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
import { AppRoutingModule } from './app.routing.module';
import { ClientModule} from './module/noticias.module';

@NgModule({
  imports:[ BrowserModule, InfiniteScrollModule,BrowserAnimationsModule,
        AppRoutingModule, ClientModule ],
  declarations: [ AppComponent, ],
  bootstrap: [ AppComponent ]
})
export class AppModule { }`

I don't understand what i'm doing wrong, don't have problem with these because I use the ClientModule and works.

**Comment by Jvick13** [comment]
OMG! Hahahaha, Thanks @gregduffield I finally realized what's the matter, how dumb!

**Comment by Nikitas-io** [comment]
> Had the same issue, make sure that you are also exporting the Module as well as Importing in your module.ts

Can somebody please explain what this means? Export WHICH module from where and Import WHICH module?

**Comment by ErEngineering** [comment]
Hello @gregduffield, @Jvick13 
Could you explain the solution please, i **have the same problem** and i don't understand your explanations

**Comment by gii-rish** [comment]
Import `InfiniteScrollModule` in that module where you are using the infinite scroll.

**Comment by ZaidPathan** [comment]
Installing v10.x worked for me as my project Angular version is 12.x.
